src/common/components/dataTable.js

```diff
@@ -0,0 +1,72 @@
+import React, { Fragment } from "react";
```

Comment:
I think it makes sense to call this component Observations as that is how it is named in the avni-client app.